<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmPopup</class>
 <widget class="QWidget" name="frmPopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>消息</string>
  </property>
  <property name="windowIcon">
   <iconset resource="main.qrc">
    <normaloff>:/image/main.png</normaloff>:/image/main.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12pt &quot;宋体&quot;;</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>361</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="btnReset">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>85</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>软件复位</string>
    </property>
   </widget>
   <widget class="QLabel" name="labHostName">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;黑体&quot;;
color: rgb(0, 170, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labTriggerContent">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;黑体&quot;;
color: rgb(0, 170, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/sound.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>btnReset</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections/>
</ui>
